editor/src/clj/editor/lua.clj

```diff
@@ -32,12 +32,46 @@
 (set! *warn-on-reflection* true)
 
 (def ^:private docs
-  ["base" "bit" "buffer" "b2d" "b2d.body" "builtins" "camera" "collectionfactory"
-   "collectionproxy" "coroutine" "crash" "debug" "factory" "go" "gui" "graphics"
-   "html5" "http" "image" "io" "json" "label" "liveupdate" "math" "model" "msg"
-   "os" "package" "particlefx" "physics" "profiler" "render" "resource"
-   "socket" "sound" "sprite" "string" "sys" "table" "tilemap" "timer" "vmath"
-   "window" "zlib" "types"])
+  ["lua_base.doc_h" "lua_coroutine.doc_h" "lua_debug.doc_h"
```

Comment:
The api ref files are no longer named from their namespace but from the actual filename. An improvement would be to look for all .sdoc files with `@language Lua`